settings.bash

```diff
@@ -0,0 +1 @@
+
```

Comment:
This should not be there! :)

Reply:
It's because i put settings.bash in a `files()` in meson.build - this will error if the file does not exist. So I decided that the "clean state" of the repository would be an empty file instead of a non-existing file. At least if this shouldn't be here the meson.build should be changed.

Reply:
Ok, I would rather have a work-around for this in meson to not confuse what `settings.bash` should be used for. :)